include/exadg/fluid_structure_interaction/precice/exadg_coupling.h

```diff
@@ -38,13 +38,16 @@ template<int dim, int data_dim, typename VectorizedArrayType>
 class ExaDGCoupling : public CouplingBase<dim, data_dim, VectorizedArrayType>
 {
 public:
+  static unsigned int const rank =
+    (data_dim == 1) ? 0 : ((data_dim == dim) ? 1 : dealii::numbers::invalid_unsigned_int);
```

Comment:
What about moving this into a helper function? The same lines of codes are below. Also: what is `dealii::numbers::invalid_unsigned_int`?

Reply:
make sense

Reply:
see https://github.com/exadg/exadg/pull/230/commits/729bfceb81cd475643b51e97bec9fcac03537352